rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -190,6 +197,78 @@ def _repr_mimebundle_(self, **kwargs: dict) -> tuple[dict, dict] | None:  # type
     def _repr_keys(self):  # type: ignore[no-untyped-def]
         return self._viewer._repr_keys()
 
+    def update_panels(
```

Comment:
A "partial update" API felt the most intuitive to me. This means that we need a way to differentiate between "don't update this panel state" (`None`) and "please reset this panel state" (`default`).